packages-internal/scripts/generate-llms-txt/src/processApi.ts

```diff
@@ -93,7 +97,11 @@ function htmlToMarkdown(html: string): string {
     // Convert <code> to backticks
     .replace(/<code>([^<]+)<\/code>/gi, '`$1`')
     // Convert <a> to markdown links
-    .replace(/<a\s+href="([^"]+)">([^<]+)<\/a>/gi, '[$2]($1)');
+    .replace(/<a\s+href="([^"]+)">([^<]+)<\/a>/gi, (match, href, text) => {
+      // Add host if provided and href is relative
+      const url = host && href.startsWith('/') ? `${host}${href}` : href;
```

Comment:
Avoid string manipulation for urls
```suggestion
      const url = host && href.startsWith('/') ? new URL(href, host).href : href;
```